packages/babel-helper-define-polyfill-provider/src/utils.ts

```diff
@@ -1,18 +1,15 @@
-// @flow
-
-import * as babel from "@babel/core";
-const { types: t, template } = babel.default || babel;
-import type NodePath from "@babel/traverse";
+import { types as t, template } from "@babel/core";
+import type { NodePath } from "@babel/traverse";
```

Comment:
We might have to add `@types/babel__traverse` to this package's `devDependencies`, becuase the packages published from the babel/babel repo don't include yet type definitions (except for `@babel/types` and `@babel/parser`). We'll start publishing them in Babel 8.

Reply:
So when I did a "Go to Definition" in VS Code for `NodePath`, it did take me to `@types/babel__traverse` in `node_modules`. So I believe it may already be there due to a transitive dependency.

I did try `yarn add -dev @types/babel__traverse` and that only fixed 3 type errors 😕